components/VaultAprSources/index.tsx

```diff
@@ -12,18 +12,29 @@ function toAprRange(apy: number, minApr?: number, maxApr?: number) {
 }
 
 function VaultAprSources({ vault }: Props): JSX.Element {
-  const { apr, minApr, maxApr } = vault;
-  const currentYieldDisplay = toAprRange(apr, minApr, maxApr);
+  const { apr } = vault;
+
+  const currentYieldDisplay = toAprRange(
+    apr.baseYield,
+    apr.minYield,
+    apr.maxYield,
+  );
 
   let yieldDisplay: React.ReactNode;
-  if (vault.sourcesApy.length > 0) {
-    yieldDisplay = vault.sources.map((s) => (
+  if (vault.apr.sources.length > 0) {
```

Comment:
apy.sources

Reply:
its in other spots too

Reply:
i was bit confused, coz comp called AprSources >_>

Reply:
but makes sense, fixed